back_end/src/apis/friend.py

```diff
@@ -0,0 +1,75 @@
+from flask_cors import CORS
+from flask import Blueprint, request, jsonify
+from flask_login import login_required, current_user
+
+from ..models.friend import Friend
+from ..models.user import User
+
+friend_api_bp = Blueprint('friend_api', __name__)
+CORS(friend_api_bp, supports_credentials=True)
+
+
+@friend_api_bp.route('/request_friend', methods=['POST'])
+@login_required
+def request_friend():
+    req_data = request.get_json()
+    # u1_id = req_data.get('requestor_id')
+    u2_id = req_data.get('reciever_id')
```

Comment:
here you put reciever_id instead of rec**ei**ver_id, everywhere else looks like its fine. This doesn't break anything, its just for consistency when front end passes in POST request

Reply:
Yes! Too many typos